packages/grid/x-grid-modules/src/components/styled-wrappers/grid-root.tsx

```diff
@@ -154,6 +154,13 @@ export const RootStyle = styled.div<GridRootProps>`
           &.checkbox-selection-header-cell .checkbox-input {
             padding: 12px;
           }
+          .dragging {
+            background: gray;
```

Comment:
cursor?

Reply:
Btw I checked with the other grid libs, they don't have a specific cursor, like `grab` for dragging, they use `pointer` and the `material-col-cell` already has it.